sdcm/sct_config.py

```diff
@@ -847,6 +847,18 @@ class SCTConfiguration(dict):
 
         dict(name="append_scylla_setup_args", env="SCT_APPEND_SCYLLA_SETUP_ARGS", type=str,
              help="More arguments to append to scylla_setup command line"),
+
+        dict(name="stress_cdclog_reader_cmd", env="SCT_STRESS_CDCLOG_READER_CMD",
+             type=str,
+             help="""cdc-stressor command to read cdc_log table.
+                    You can specify everything but the -node , -keyspace, -table, parameter, which is going to
+                    be provided by the test suite infrastructure.
+                    multiple commands can passed as a list"""),
+
+        dict(name="update_cdclog_reader_stats", env="SCT_UPDATE_CDCLOG_READER_STATS",
```

Comment:
isn't that called upload ? or something a bit like `store_results_in_elasticsearch`

Reply:
The problem is that cdc stress tool for that moment don't correclty count latency. So i decided to add new parameter which allow to disable the store only cdc-stress tool stats in ES and this doesn't allow to calculate this stats in performance result calculation.
So only stats from c-s tool will be used in regression stats.
Once the cdcstressor tool will be fixed with this,  it will easlier to enable the its stats in perf regression